yt/visualization/volume_rendering/scene.py

```diff
@@ -204,6 +204,16 @@ def render(self, camera=None):
         >>> sc = yt.create_scene(ds)
         >>> # Modify camera, sources, etc...
         >>> im = sc.render()
+        >>> sc.save(sigma_clip=4.0,render=False)
+
+        Altneratively, if you do not need the image array, you can just call
```

Comment:
This is a nice enhancement to the docs showing how to use .save() both ways. One way that this could be enhanced is by showing an interim operation on `im` in the example above, though I'm not positive of what a good thing to do would be here.

Reply:
I'm not sure either. I have ideas, but they seem too limited in scope to be useful. For example, you could plot an image histogram of the RGB channels from `im` (something like this https://en.wikipedia.org/wiki/Image_histogram#/media/File:SunHistacp.jpg ?) or you could apply an image processing function like one of the exposure adjustments in https://scikit-image.org/docs/stable/api/skimage.exposure.html to the image array but both of these feel too involved (and limited in usefulness) for a docstring example?